requirements.txt

```diff
@@ -0,0 +1,5 @@
+aenum==2.0.8
+
+# Documentation
+Sphinx==1.6.5
```

Comment:
can you please make a separate dev requirements file and have setup.py parse these? The dependencies were installed before via setup.py but maybe having requirements.txt is easier.

Reply:
I'm not sure we need two requirements files (unless I am missing something). The aenum module is required to run the module, and sphinx for the docs (I guess we need them for both dev and prod so that we can build and test them locally).

Maybe we can just have one requirements file and make setup.py parse that?

Reply:
Nvm - we can keep sphinx in requirements-dev.txt so it doesn't install on setup.py install.